src/test/resources/test.template.properties

```diff
@@ -10,3 +10,6 @@ test.persistence.timeout=4
 # Please read the snapshot testing documentation if you are not yet familiar with it, and use with care.
 # Remember to set back to false when done and rerun the test(s).
 test.snapshot.update=false
+
+# This is the port to connect with local datastore emulator.
```

Comment:
The wording implies that the emulator is already there, but it is not the case. In the setup, we're spinning up the emulator on this port. Probably more accurate if it says ~to connect with~ *where* local datastore emulator *will be instantiated*.
Also need to add word of caution that it must be a free port.